samples/tetris/build.gradle

```diff
@@ -60,10 +62,16 @@ kotlin {
     }
 }
 
-if (hostPreset == kotlin.presets.mingwX64) {
+if ((hostPreset == kotlin.presets.mingwX86) || (hostPreset == kotlin.presets.mingwX64)) {
     tasks.create('compileWindowsResources', Exec) {
+        def windresDir = null
+        if (hostPreset == kotlin.presets.mingwX86)
+            windresDir = "${MPPTools.kotlinNativeDataPath()}/dependencies/msys2-mingw-w64-i686-gcc-7.4.0-clang-llvm-6.0.1/bin"
```

Comment:
Ugly :(. Maybe at least smth like `fun targetToolchainPath(hostPreset)`.

Reply:
Another _bad_ function for _MPPTools_?

Reply:
I would suggest to add this to K/N Samples Wishlist but leave it as is for now.